tutorials/appengine-ruby-rails-activejob-sidekiq.md

```diff
@@ -0,0 +1,285 @@
+   respond to the health check ping.
+
+1. Deploy both services to App Engine
+
+        gcloud app deploy app.yaml workers.yaml
+
+## Verify your background queuing works
+
+1. In the Cloud Platform Console, go to the
+   **[App Engine Services](https://console.cloud.google.com/appengine/services)** page. Locate the service that is
+   running your background workers (if option A, it should be the *default* service, if option B, it should be
+   the *workers* service). Click Tools -> Logs for that service.
+
+1. On the Logs dashboard, click the Play icon to start streaming the logs.
+
+1. In a separate window, navigate to your deployed Rails application at:
+
+        https://[YOUR_PROJECT_ID].appspot.com/say/Jeff
+
+   Be sure to replace `[YOUR_PROJECT_ID]` with your Google Cloud Platform project ID.
+
+1. Navigate back to the Logs dashboard and you should see a logging statement like:
+
+        13:13:52.000 Hello, Jeff from ActiveJob!
+
+Congratulations, you have successfully set up background job processing on Google App Engine with Sidekiq.
+
+## Cleaning up
+
+After you've finished this tutorial, you can clean up the resources you created on Google Cloud Platform
+so you won't be billed for them in the future. The following sections describe how to delete or turn off these
+resources.
+
+### Deleting the project
+
+The easiest way to eliminate billing is to delete the project you created for the tutorial.
+
+To delete the project:
+
+1. In the Cloud Platform Console, go to the **[Projects](https://console.cloud.google.com/iam-admin/projects)** page.
+1. Click the trash can icon to the right of the project name.
+
+**Warning**: Deleting a project has the following consequences:
+
+If you used an existing project, you'll also delete any other work you've done in the project.
+You can't reuse the project ID of a deleted project. If you created a custom project ID that you plan to use in the future, you should delete the resources inside the project instead. This ensures that URLs that use the project ID, such as an appspot.com URL, remain available.
+
+### Deleting App Engine services
+
+To delete an App Engine service:
+
+1. In the Cloud Platform Console, go to the **[App Engine Services](https://console.cloud.google.com/appengine/services)** page.
+1. Click the checkbox next to the service you wish to delete.
+1. Click the Delete button at the top of the page to delete the service.
+
+If you are trying to delete the *default* service, you cannot. Instead:
+
+1. Click on the number of versions which will navigate you to App Engine Versions page.
+1. Select all the versions you wish to disable and click the Stop button at the top of the page. This will free
+   all of the Google Compute Engine resources used for this App Engine service.
+
+## Next steps
+
+* Explore [the example code](https://github.com/chingor13/rails-background-jobs) used in this tutorial.
```

Comment:
Will you be writing a README for the project that directs users to the community article? 😄